stdlib/public/Concurrency/TaskCancellation.swift

```diff
@@ -51,13 +51,7 @@ extension Task {
   ///
   /// - SeeAlso: `checkCancellation()`
   public var isCancelled: Bool {
```

Comment:
```suggestion
  @_transparent public var isCancelled: Bool {
```

Making this `@_transparent` will fix this more comprehensively: existing clients running on a newer OS will get the fix automatically (from the ABI-stable version), and clients recompiled with a newer _Concurrency module will inline the fixed version.